### ISSUE TITLE
Create passport-rp.pem contents

### ISSUE THREAD

**Issue by afroDC** [GluuFederation/docker-config-init#5]
We're missing the creation of `passport-rp.pem` from configuration creation.

This command would presumably create it:

```
keytool -keystore passport-rp.jks -keypasswd passport_rp_client_jks_pass -alias passport_rp_client_cert_alias -exportfile passport_rp_client_cert_fn
```

**Comment by iromli** [comment]
Patches available at 291003ff0dd47a6132babe3c89676608e7d65b19

**Comment by iromli** [comment]
Fixed in `3.1.2` branch
